packages/phone-number-privacy/src/common/identity.ts

```diff
@@ -1,15 +1,26 @@
 import { AttestationsWrapper } from '@celo/contractkit/lib/wrappers/Attestations'
+import { verifySignature } from '@celo/utils/lib/signatureUtils'
+import { Request } from 'firebase-functions'
 import config from '../config'
 import { getContractKit } from '../web3/contracts'
 import logger from './logger'
 
 /*
  * Confirms that user is who they say they are and throws error on failure to confirm.
+ * Authorization header should contain the EC signed body
  */
-export function authenticateUser() {
+export function authenticateUser(request: Request): boolean {
   logger.debug('Authenticating user')
-  // TODO [amyslawson]
-  return
+
+  // https://tools.ietf.org/html/rfc7235#section-4.2
+  const messageSignature = request.get('Authorization')
```

Comment:
can we update the curl examples in index.js to include what the header should look like?

Reply:
Updated!